cmd/derper/derper.go

```diff
@@ -216,6 +217,15 @@ func main() {
 	}))
 	debug.Handle("traffic", "Traffic check", http.HandlerFunc(s.ServeDebugTraffic))
 
+	// Longer lived DERP connections send an application layer keepalive.
+	// Note if the keepalive is hit, the user timeout will take precedence over
+	// keepcnt, so the probe if unanswered will take effect promptly, this is
+	// less tolerant of high loss, but high loss is unexpected.
+	lc := net.ListenConfig{
+		Control:   ktimeout.UserTimeout(10 * time.Second),
+		KeepAlive: 10 * time.Minute,
```

Comment:
Where do we actually define our application-level keepalive interval? I'm trying to check these against that value and am failing to find it 😵‍💫

Reply:
derp_server.go `keepAliveTick`

Reply:
https://github.com/tailscale/tailscale/blob/main/derp/derp_server.go#L1443

Reply:
Thanks 👍